### PR TITLE
more fixes in quadratic forms

### PR DESCRIPTION
**Pull request by fchapoton**
This is fixing a few code and doc details in two files inside quadratic forms.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/quadratic_forms/genera/genus.py [hunk 1]
```diff
@@ -298,13 +298,13 @@ def _blocks(b, even_only=False):
...
                 b1 = copy(b)
-                b1[2] = s[0]*(-1)**(rk//2 -1) % 8
+                b1[2] = s[0]*(-1)**(rk // 2 - 1) % 8
                 b1[3] = 1
                 b1[4] = s[1]
                 blocks.append(b1)
             for s in [(1, 4), (5, 0)]:
                 b1 = copy(b)
-                b1[2] = s[0]*(-1)**(rk//2 - 2) % 8
+                b1[2] = s[0]*(-1)**(rk // 2 - 2) % 8
```
Comment (mkoeppe) [inline src/sage/quadratic_forms/genera/genus.py:307]: PEP8 requires spaces around the outermost arithmetic operators only

Comment (fchapoton) [reply src/sage/quadratic_forms/genera/genus.py:307]: this is one instance of pycodestyle warning E226

Comment (mkoeppe) [reply src/sage/quadratic_forms/genera/genus.py:307]: See also https://github.com/PyCQA/pycodestyle/issues/248

**Review by tscrim — APPROVED** [review]
LGTM.

**Pull request merged**